Tests/Data/Parabolic/T/3D_2U_BHE/3D_2U_BHE.prj

```diff
@@ -60,15 +60,15 @@
                 <phase>
```

Comment:
Some parts of the project files are not properly formatted. To correct the formating you can use xmlstarlet.
```
xmlstarlet fo -s 4 input.prj > output.prj
mv output.prj input.prj
````

Reply:
All project files have been reformatted using xmlstarlet.